### ISSUE TITLE
getting a csv or text file for existing resources

### ISSUE THREAD

**Issue by iahmad-khan** [pdellaert/vSphere-Python#12]
Hi , 

It would be very helpful if we can get a list or csv of exisitng vms , with thier datasotres , resource pools , and folders etc. 

so that we can re-use and change that information in creating new vms.

**Comment by ibenrodriguez** [comment]
Yes! I'd like to +1 this request. We have a windows app called RVTOOLs that does this and it's be great to have this available via CLI / scripts.
